pyfr/plugins/base.py

```diff
@@ -6,6 +6,7 @@
 
 import numpy as np
 from pytools import prefork
+import subprocess
```

Comment:
Must use prefork for calling scripts to avoid crashes with certain MPI libraries.

Reply:
Yes true, sorry for that I was just doing some tests